Specs/Scene/ModelSpec.js

```diff
@@ -2330,6 +2336,49 @@ defineSuite([
         });
     });
 
+    it('loads a glTF with KHR_draco_mesh_compression extension', function() {
```

Comment:
Do these tests have code coverage for all the new code?

Do we test both with and without quantization?

Reply:
Ah, I did not realize that quantization would be a separate PR.  I was under the impression that Draco was "done."  Let's please finish Draco with quantization before moving on to the GLSL extension.